doc/modules/tree.rst

```diff
@@ -534,16 +534,57 @@ Mean Absolute Error:
 
 where :math:`X_m` is the training data in node :math:`m`
 
+
+.. _minimal_cost_complexity_pruning:
+
+Minimal Cost-Complexity Pruning
+===============================
+
+Minimal cost-complexity pruning is an algorithm used to prune a tree, described
+in Chapter 3 of [BRE]_. This algorithm is parameterized by :math:`\alpha\ge0`
```

Comment:
It might be worth adding a small note to say that this is one method used to avoid over-fitting in trees.